acceptance/ui/features/hosts.feature

```diff
@@ -18,15 +17,204 @@ Feature: Host Management
+      And I sort by "Active" in descending order
+    Then the "Active" column should be sorted with active hosts on top
+
+  @login-required
+  Scenario: Test descending resource pool sort
+    When I am on the hosts page
+      And I sort by "Resource Pool" in descending order
+    Then the "Resource Pool" column should be sorted in descending order
+
+  @login-required
+  Scenario: Test ascending resource pool sort
+    When I am on the hosts page
+      And I sort by "Resource Pool" in ascending order
+    Then the "Resource Pool" column should be sorted in ascending order
+
+  @login-required
+  Scenario: Test descending memory sort
+    When I am on the hosts page
+      And I sort by "Memory" in descending order
+    Then the "Memory" column should be sorted in descending order
+
+  @login-required
+  Scenario: Test ascending memory sort
+    When I am on the hosts page
+      And I sort by "Memory" in ascending order
+    Then the "Memory" column should be sorted in ascending order
+
+  @login-required
+  Scenario: Test ascending CPU cores sort
+    When I am on the hosts page
+      And I sort by "CPU Cores" in ascending order
+    Then the "CPU Cores" column should be sorted in ascending order
+
+  @login-required
+  Scenario: Test descending CPU cores sort
+    When I am on the hosts page
+      And I sort by "CPU Cores" in descending order
+    Then the "CPU Cores" column should be sorted in descending order
+
+  @login-required
+  Scenario: Test ascending kernel version sort
+    When I am on the hosts page
+      And I sort by "Kernel Version" in ascending order
+    Then the "Kernel Version" column should be sorted in ascending order
+
+  @login-required
+  Scenario: Test descending kernel version sort
+    When I am on the hosts page
+      And I sort by "Kernel Version" in descending order
+    Then the "Kernel Version" column should be sorted in descending order
+
+  @login-required
+  Scenario: Test ascending CC release sort
+    When I am on the hosts page
+      And I sort by "CC Release" in ascending order
+    Then the "CC Release" column should be sorted in ascending order
+
+  @login-required
+  Scenario: Test descending CC release sort
+    When I am on the hosts page
+      And I sort by "CC Release" in descending order
+    Then the "CC Release" column should be sorted in descending order
+
+  @login-required @defaultHostPage
+  Scenario: Add a duplicate host
+    When I am on the hosts page
+      And I click the Add-Host button
+      And I fill in the Host Name field with "172.17.42.1:4979"
       And I fill in the Resource Pool field with "default"
-      And I fill in the RAM Commitment field with "100%"
+      And I fill in the RAM Commitment field with "50%"
       And I click "Add Host"
-    Then I should see "gjones-dev"
-      And I should see "default"
-      And I should see "Delete"
+    Then I should see "Error"
+      And I should see "Internal Server Error: host already exists"
```

Comment:
Please add a verification step like `And I should see an empty Hosts page` as mentioned above.